wiki/BioSQL.md

```diff
@@ -248,36 +261,49 @@ Running the unit tests
 ----------------------
 
 Because there are so many ways you could have setup your BioSQL
-database, you have to tell the unit test a few bits of information by
-editing the file Tests/setup\_BioSQL.py and filling in the following
-fields:
+database, you have to tell the unit test a few bits of information. 
+If you installed biopython using pip then the relevant [**Tests**](https://github.com/biopython/biopython/tree/master/Tests) 
+folder would not have been copied. If this is the case one can 
+perform check out (or export) using:
 
-``` python
-DBDRIVER = 'MySQLdb'
-DBTYPE = 'mysql'
+```bash
+svn export https://github.com/biopython/biopython/trunk/Tests
```

Comment:
Again, please avoid ``svn export`` and use ``git`` or a plain download from GitHub.